### ISSUE TITLE
Unclear Activity Types

### ISSUE THREAD

**Issue by IKnowLogic** [BONSAMURAIS/arborist#19]
In my perspective, Activity Types should all have informative descriptions of some activity taking place.
This is the case for many of the activities, such as **Cultivation of wheat** and **Processing of meat pigs**. These are easy to read descriptions (`rdfs:label`) of the given activity. However, not all activities have such easy to understand descriptions, for example **Wool, silk-worm cocoons**, **Meat animals nec**, **Raw milk** or **Animal products nec**. It is hard to understand what is the activity taking place here. I think this problems relates to the Bonsai classification when extracting information from Exiobase.

**Comment by mfastudillo** [comment]
Agree, although in many cases a correspondence with the NACE2 classification would provide a description. Many of the exiobase names are just the NACE2 "label"
